focus-android/app/src/main/java/org/mozilla/focus/utils/IntentUtils.kt

```diff
@@ -19,8 +19,8 @@ object IntentUtils {
      * This additional requirement improves your app's security.
      * FLAG_IMMUTABLE -> Flag indicating that the created PendingIntent should be immutable.
      */
-    val defaultIntentPendingFlags
-        get() = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+    fun defaultIntentPendingFlags(buildVersion: Int = Build.VERSION.SDK_INT): Int =
```

Comment:
Aside from the IntentUtilsTest, are there any other places we use this as a param? I don't follow the reason for this change as part of the JVM 17 upgrade. 🤔

Reply:
This is needed because we can no longer use reflection to change the `SDK_INT value`. Instead, if we can pass it as a param we can easily modify the value in tests. 
I would actually prefer to do this anywhere, even if we could have used reflection.